backend/src/validations/questionValidation.ts

```diff
@@ -0,0 +1,8 @@
+import Joi from "joi";
+
+export const createQuestionValidation = Joi.object({
+    title: Joi.string().min(3).max(100).required(),
+    type: Joi.string().valid("text", "dropdown").required(),
+    options: Joi.array().items(Joi.string())
```

Comment:
Can you put the validation that when the type is equal to dropdown the options must be required